sdk/attestation/azure-security-attestation/azure/security/attestation/_models.py

```diff
@@ -948,3 +948,16 @@ def __init__(self, message):
         super(  # pylint: disable=super-with-arguments
             AttestationTokenValidationException, self
         ).__init__(self.message)
+
+class TpmAttestationResult(object):
```

Comment:
Why was this introduced?

Reply:
This was added to match the other attest functions, such as attest_sgx_enclave, which returns a similar object.

Reply:
As mentioned, we will not break the existing method signature. But breaking changes aside - we do not use "getters" in Python.

Reply:
Removed getter